### ISSUE TITLE
how to build package on focal ?

### ISSUE THREAD

**Issue by inf0rmix** [kodi-game/game.libretro.mupen64plus-nx#13]
Hi,

just a stupid question:

how can I build the package under ubuntu/debian ?

I only find a debian/changelog.in and dpkg-buildpackage needs a debian/changelog with correct data.

**Comment by inf0rmix** [comment]
thank you very much for the info! is that the reason why e.g. mupen crashes after loading a rom ?

**Comment by garbear** [comment]
Yes, I think I found the reason and fixed it here: https://github.com/kodi-game/game.libretro/pull/80

I'll test mupen and see if the crash is fixed.

**Comment by inf0rmix** [comment]
thank you for the information. I will test it on friday
